client/main.lua

```diff
@@ -366,9 +378,15 @@ RegisterNUICallback('select', function(data, cb)
         elseif option.command then
             ExecuteCommand(option.command)
         end
+        
+        if option.openMenu then
+            currentMenu = option.openMenu
+            state.setNuiFocus(true, true, false)
+            forceNuiActive = true
```

Comment:
Wouldn't this cause a loop jif you had `openMenu = 'home'` on a root level option? It would just keep opening the home page instead of closing out altogether as would previously happen, right?

Reply:
Updated this